### PR TITLE
Feat stills overlay and masonry gutter

### PR DESCRIPTION
**Pull request by mixcas**

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -11,6 +11,12 @@ var largeImagesTriggerWidth = 700,
 
   directorShowreelVideos = $('.director-showreel-video'),
   directorShowreelLength = directorShowreelVideos.length,
+  directorShowreelVideos = $('.director-showreel-video'),
```
Comment (varffs) [inline js/main.js]: doubled?

File: js/main.js [hunk 2]
```diff
@@ -11,6 +11,12 @@ var largeImagesTriggerWidth = 700,
...
+  directorShowreelLength = directorShowreelVideos.length,
+
+  directorStills = $('.still'),
+  directorStillsLenght = directorStills.length,
```
Comment (varffs) [inline js/main.js]: typo on variable

File: js/main.js [hunk 3]
```diff
@@ -161,6 +167,12 @@ function directorInit() {
   $('.js-load-overlay-vimeo').on('click', function () {
     overlayVimeoPlayer.load($(this).data(), $(this).index());
   });
+    
+    // LOAD STILLS IN OVERLAY
+
+  $('.js-load-overlay-image').on('click', function () {
+    overlayImage.load($(this).children('img').clone(), $(this).index());
```
Comment (varffs) [inline js/main.js]: cloning the code means the image is the small size no?

File: js/main.js [hunk 4]
```diff
@@ -251,6 +276,81 @@ function directorInit() {
   });
 
 }
+// OVERLAY IMAGE
+var overlayImage = {
+
+  player: $('#image-overlay-player'),
```
Comment (varffs) [inline js/main.js]: should we refactor the terminology? not a player anymore but an image

File: js/main.js [hunk 5]
```diff
@@ -251,6 +276,81 @@ function directorInit() {
...
+
+    image.attr('width', '100%').removeAttr('height');
+
+    _this.overlay.fadeIn(fastAnimationSpeed).data('now-playing', postIndex);
+    _this.player.html(image);
+
+    // use fixHeight after animations
+      _this.fixHeight();
+    
+    $(window).resize(function() {
```
Comment (varffs) [inline js/main.js]: probably not needed anymore as there is no longer an opening animation from the player

File: js/main.js [hunk 6]
```diff
@@ -251,6 +276,81 @@ function directorInit() {
...
+      this.load(image, prevousIndex);
+    }
+  },
+
+  close: function () {
+    this.overlay.fadeOut(fastAnimationSpeed).data('now-playing', '');
+    this.player.html('');
+  },
+
+  fixHeight: function () {
```
Comment (varffs) [inline js/main.js]: couldn't all this functionality be fixed for an image just with css? or is it not worth bothering?

Comment (mixcas) [inline js/main.js]: like use the original functions? i tried that, this was way easier

File: js/main.js (hunk 3 above)
Comment (mixcas) [inline js/main.js]: no cuz img with src set! 💃

Comment (mixcas) [inline js/main.js]: i guess this need WP update to work no?

**Comment by mixcas** [comment]
assigned to u @varffs , doesn't need much checking

File: js/main.js (hunk 3 above)
Comment (varffs) [inline js/main.js]: hmm but the src-set is based off the size specified by that image. which is 'still' or something and not a particularly large crop right?

File: css/site.css [hunk 7]
```diff
@@ -700,19 +700,19 @@ iframe {
   padding-top: 16px;
   padding-right: 32px;
 }
-#vimeo-player,
+#vimeo-max-player,
```
Comment (varffs) [inline css/site.css]: ???

**Comment by mixcas** [comment]
@varffs !

**Comment by varffs** [comment]
you can close if happy with those edits

**Pull request merged**